packages/datagateway-download/src/downloadStatus/downloadStatusTable.component.tsx

```diff
@@ -300,59 +300,56 @@ const DownloadStatusTable: React.FC<DownloadStatusTableProps> = (
             actions={[
               function DownloadButton({ rowData }: TableActionProps) {
                 const downloadItem = rowData as FormattedDownload;
-                const isDownloadable = (downloadItem.transport as string).match(
-                  /https|http/
-                )
+                const isHTTP = downloadItem.transport.match(/https|http/)
                   ? true
                   : false;
 
+                const isComplete =
+                  downloadItem.status === t('downloadStatus.complete')
+                    ? true
+                    : false;
+
+                const isDownloadable = isHTTP && isComplete;
+
                 return (
                   <BlackTooltip
                     title={
-                      t('downloadStatus.download_disabled_tooltip', {
-                        transport: downloadItem.transport,
-                      }) as string
+                      !isHTTP
+                        ? (t(
+                            'downloadStatus.non_https_download_disabled_tooltip',
+                            { transport: downloadItem.transport }
+                            // for some reason it can't infer these types on its own
```

Comment:
I'm guessing because the `title` is `NonNullable`, which excludes null and undefined and `useTranslation` can return undefined? 🤷 The solution works though!

Reply:
If I called the translation function outside the return and stored it in a variable then I could use that, hence I don't know why it didn't like it 😅

Plus the previous value of `title` was also a translation....